tests/template/template_test.php

```diff
@@ -603,6 +603,53 @@ public function test_more_alter_block_array()
 		$this->assertEquals($expect, str_replace(array("\n", "\r", "\t"), '', $this->display('test')), 'Ensuring S_NUM_ROWS is correct after modification');
 	}
 
+	public function test_delete_alter_block_array()
+	{
+		$this->template->set_filenames(array('test' => 'loop_nested.html'));
+
+		$this->template->assign_var('TEST_MORE', true);
+
+		// @todo Change this
```

Comment:
Copy & paste from https://github.com/javiexin/phpbb/blob/136c74bd1c3d2ef1f22055d9bb14e6f524d41433/tests/template/template_test.php#L558
and https://github.com/javiexin/phpbb/blob/136c74bd1c3d2ef1f22055d9bb14e6f524d41433/tests/template/template_test.php#L581
The reason/explanation might be this one https://github.com/javiexin/phpbb/blob/136c74bd1c3d2ef1f22055d9bb14e6f524d41433/tests/template/template_test.php#L19-L21